es-vue-base/src/lib-components/EsHorizontalList.vue

```diff
@@ -35,6 +42,13 @@
                     active: index === activeIndex,
                 }"
                 @click="dotClick(index)" />
+            <es-button
+                v-if="arrows"
+                variant="link"
+                class="text-black d-none d-lg-block p-0 ml-200"
+                @click="arrowClick('right')">
+                <icon-chevron-right />
+            </es-button>
```

Comment:
Can we throw a `<span class="sr-only">Next</span>` in here for accessibility purposes, so the function if this button is clear to screen readers?